src/workerd/api/node/zlib-util.c++

```diff
@@ -438,18 +444,20 @@ void ZlibContext::setOutputBuffer(kj::ArrayPtr<kj::byte> output) {
   stream.avail_out = output.size();
 }
 
-jsg::Ref<ZlibUtil::ZlibStream> ZlibUtil::ZlibStream::constructor(ZlibModeValue mode) {
-  return jsg::alloc<ZlibStream>(static_cast<ZlibMode>(mode));
+template <typename CompressionContext>
+jsg::Ref<ZlibUtil::CompressionStream<CompressionContext>> ZlibUtil::CompressionStream<
+    CompressionContext>::constructor(ZlibModeValue mode) {
+  return jsg::alloc<CompressionStream>(static_cast<ZlibMode>(mode));
 }
 
 template <typename CompressionContext>
-CompressionStream<CompressionContext>::~CompressionStream() noexcept(false) {
+ZlibUtil::CompressionStream<CompressionContext>::~CompressionStream() {
```

Comment:
Why did you take the `noexcept(false)` off this signature? It should be retained.

Reply:
It's producing the following error that requires some hacky solution to get around it:

```
src/workerd/api/node/zlib-util.h:260:5: error: exception specification is not available until end of class definition
  260 |     ~CompressionStream() noexcept(false);
```